ibis/expr/operations/core.py

```diff
@@ -183,5 +184,9 @@ class Argument(Value):
     shape: ds.DataShape
     dtype: dt.DataType
 
+    @attribute
+    def param(self) -> str:
```

Comment:
I am confusing it with `ops.ScalarParameter` based in the attribute name. By the way this seems to be an implementation detail of the sql backends (in particular duckdb), could you implement the "unique parameter name" generation in the compiler instead?

Reply:
I'd rather not.

Reply:
I have to do it for each backend individually then, and that's not worth the benefit.

Reply:
Okey then, keep it :)